docs/specs/0024-align-in-app-brand-to-design-system.md

```
@@ -0,0 +1,81 @@
+# SPEC: feat(brand): align the in-app mark and copy to the design system
+
+## Problem
```

markdownlint-cli2 (0.23.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)